src/Model/Conversation.php

```diff
@@ -17,13 +17,14 @@ class Conversation
 	 * These constants represent the parcel format used to transport a conversation independently of the message protocol.
 	 * It currently is stored in the "protocol" field for legacy reasons.
 	 */
-	const PARCEL_UNKNOWN            = 0;
+	const PARCEL_ACTIVITYPUB        = 0;
```

Comment:
There is currently a weighting of protocol "quality". There can be the case that the same message url arrives over different protocols. Then I keep the one with the lower number. Since I would like to switch to AP in some time in the future, it does make sense to make this the number 0.

Reply:
What about the existing conversation rows that had `protocol = 0`?

Reply:
They will grow out of this in a few days, since we don't held data very long in that table.